### DISCUSSION TITLE
Erweiterung Heizstab auxPower

### DISCUSSION THREAD

**Discussion by Michel83** [ha-puzzles/evcc-grafana-dashboards#36] [Ideas]
Hi, besteht interesse an einer Erweiterung für ein oder mehrere auxPower meter.

Ist vielleicht interessant für Leute mit einen Heizstab weil die Energie jetzt einfach im Hausverbrauch verschwindet.

Da ich mich ja jetzt ein wenig mit dem Script beschäftigt habe würde ich versuchen die dashboards und das Script dafür anzupassen.

Grüsse

**Comment by cschlipf** [comment]
Ich würde gerne das Basis Dashboard so allgemein wie möglich halten. Meine Annahme wäre, dass EVCC primär zum Überschussladen verwendet wird und 2 Wallboxen für viele Privathäuser entweder schon normal, oder noch normal werden wird.

Ist aber nur meine derzeitige Annahme, die auch nur auf meinen eigenen Voraussetzungen basiert. Ich lasse mich gerne von Feedback umstimmen.

Generell können wir hier aber nicht jeden Spezialfall abdecken und alles was hier zu finden ist sollte auch nur als Starthilfe gesehen werden für eigene Erweiterungen.

Du kannst aber sehr gerne hier in einem Unterordner eine weitere andere Variante als Pull Request hinzufügen.
